### ISSUE TITLE
test_variable_ids.py : remove temporary test data?

### ISSUE THREAD

**Issue by dschien** [dschien/eam-data-tools#10]
`test_variable_ids.py`
seems to create copies of
```
existing_ids_copy.xlsx
no_existing_ids_copy.xlsx
some_existing_ids_copy.xlsx
```

why are the temp copies in CSV? Can they not be in gitignore?

**Comment by theo-xir** [comment]
The three _copy files are not created during the tests but rather needed for the tests to run. The code run by the test will alter the non _copy file, then assert the two files have the exact correct differences (with ids being assigned) and then reset the non _copy file to the starting state by making it equal to the _copy file.

**Comment by theo-xir** [comment]
I could probably create the copies at the start of execution and delete them at the end to prevent extra files if that's preferable.

**Comment by dschien** [comment]
The main thing for me is that the temp files are not checked into git.

You can also create files in a temp directory.


import os
import tempfile

with tempfile.TemporaryDirectory() as tmp_dir:
    # generate some random files in it
     files_in_dir = os.listdir(tmp_dir)

________________________________

**Comment by theo-xir** [comment]
This should now be resolved as of commit f581cd58774764f1b23a7463ed8234fc982f1f5e, there are no temp files and running the tests does not result in any changes git detects
